pkg/controller/migplan/registry.go

```diff
@@ -87,6 +100,7 @@ func (r ReconcileMigPlan) ensureMigRegistries(plan *migapi.MigPlan) error {
 }
 
 // Ensure the credentials secret for the migration registry on the specified cluster has been created
+// If the secret is updated, it will return delete the imageregistry reesources
```

Comment:
`reesources` -> `resources`

Also, this sentence looks like it has an extra word in it.